main/download.js

```diff
@@ -54,7 +60,9 @@ function listenForPDFDownload (ses) {
   ses.webRequest.onHeadersReceived(function (details, callback) {
     if (details.resourceType === 'mainFrame' && details.responseHeaders) {
       var typeHeader = details.responseHeaders[Object.keys(details.responseHeaders).filter(k => k.toLowerCase() === 'content-type')]
-      if (typeHeader instanceof Array && typeHeader.filter(t => t.includes('application/pdf')).length > 0 && details.url.indexOf('#pdfjs.action=download') === -1) {
+      var attachment = isAttachment(details.responseHeaders[Object.keys(details.responseHeaders).filter(k => k.toLowerCase() === 'content-disposition')])
```

Comment:
Learned something new here :) I didn't know you could use an array to look up an object property like this, but I guess it works:
```
var x = {a: 1}
var y = x[["a"]] // = 1
```
It even works with nested arrays:
```
var y = x[[[[[["a"]]]]]] // = 1